source/_integrations/plugwise.markdown

```diff
@@ -69,6 +70,12 @@ For a thermostat, the active schedule can be deactivated or reactivated via the
 
 Auto means the schedule is active, Heat means it's not active. The active thermostat schedule can be changed via the connected thermostat select-entity. Please note: that only schedules that have two or more schedule points will be shown as select options.
 
+## Note
+
+{% note %}
+From Home Assistant 2024.12.0 onwards, climate entities connected to your Adam will no longer be shown as device-based. They will be shown as zone-based. This "mistake" was recently discovered and corrected.
```

Comment:
We don't keep changelogs in our documentation. Documentation should reflect the current state of affairs.